### DISCUSSION TITLE
who... it runs on obadooga text-web-ui

### DISCUSSION THREAD

**Discussion by kalle07** [erew123/alltalk_tts#463] [General]
all in all okay !

i dont understand how is that RVC models ?

and where i get those other models and where i copy them exactly ?!?

**Comment by erew123** [comment]
If you cannot find the information in the built in documentation, please read the WIKI https://github.com/erew123/alltalk_tts/wiki/RVC-(Retrieval%E2%80%90based-Voice-Conversion)

**Reply by kalle07** [reply]
i see ... but in most models only 
*.pth
*.index

no json is that a must have ?

and if not 
iv create 
alltalk_tts\models\rvc_voices\
-model1
and copy
*.pth and *.index
the files mostly aviable in the list
press update button
but onle the *.wav list is updated
RVC still disabled

**Reply by erew123** [reply]
Please read the setup section https://github.com/erew123/alltalk_tts/wiki/RVC-(Retrieval%E2%80%90based-Voice-Conversion)#setup

Folder creation and download of the core rvc files is automatic when you enable rvc in the global settings

**Reply by kalle07** [reply]
oh i must start the other UI ^^
Gradio Light: http://127.0.0.1:7852

ok it works all fine
iv downloaded some voices from the list , all ok
iv trained some, all okay  ;)
